spec/system/signup/index.spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'rails_helper'
+
+RSpec.describe 'Signup feature' do
+	it 'should create new user successfully' do
+		visit new_user_registration_path
+		expect(page).to have_content('Sign up new account')
+
+		fill_in('user_full_name', with: 'John Doe')
+		select "ASHA", :from => "user_role"
```

Comment:
These lines are repeating in every test iteration. Can we use [`before(:each)`](https://relishapp.com/rspec/rspec-core/v/2-0/docs/hooks/before-and-after-hooks) here